apps/apps-backend/src/features/features.controller.ts

```diff
@@ -158,6 +161,9 @@ export class FeaturesController {
                         [Network.Custom]: false,
                     },
                 },
+                [Feature.FixedGasPrice]: {
```

Comment:
same comment as above

Reply:
Done in [4658bb3](https://github.com/iotaledger/iota/pull/5847/commits/4658bb3d175361675ee6842f4b1705c360a506ea)